blog/2022-02-28-temporal-rest.md

```diff
@@ -48,13 +48,14 @@ To execute an instance of `counterWorkflow`, use `WorkflowClient` to start the W
 ```ts
 import { WorkflowClient } from '@temporalio/client';
 import { counterWorkflow } from './workflows';
+import crypto from 'crypto';
 
 async function run() {
   const client = new WorkflowClient();
 
   const handle = await client.start(counterWorkflow, {
     taskQueue: 'tutorial',
-    workflowId: 'wf-id-' + Math.floor(Math.random() * 1000),
+    workflowId: 'workflow-' + crypto.randomUUID(),
```

Comment:
I think the workflow id should come from some business id. What is the valid business id in this case?

Reply:
There isn't an existing business ID. This use case is just "start a run of the counter workflow from a command-line script for example purposes". Perhaps:

```suggestion
    // Usually, we use a business ID. In this example, we don't have one, so we generate an ID.
    workflowId: 'counter-' + crypto.randomUUID(),
```